spec/vehicle_factory_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'spec_helper'
+
+RSpec.describe VehicleFactory do
+  it "is an instance of the vehicle factory class" do
+    factory = VehicleFactory.new
+    expect(factory).to be_an_instance_of(VehicleFactory)
+  end
+
+  it "can create vehicle objects from an external database" do
+    @wa_ev_registrations = DmvDataService.new.wa_ev_registrations
+    factory = VehicleFactory.new
+    output = factory.create_vehicles(@wa_ev_registrations)
+    expect(output).to be_an(Array)
+    expect(output.first).to be_a(Vehicle)
+  end
```

Comment:
Again, we can go a little deeper here and check that all the attributes we expect to be filled in are not nil and are the correct data_type.